docs(feat): Quickspin draft WIP (#1590)
armory/docs: content/en/armory-enterprise/quickspin/quickstart-quick-spin.md

---
title: Quick Spin Quick Start 
linkTitle: "Start using Quick Spin"
no_list: true
weight: 5
description: >
  Use this tutorial to lean how to create and manage a simple Spinnaker workflow.
exclude_search: true
---
## Abstract

> This tutorial demonstrates how to:

Reviewer: Instead of "Abstract", how about "Objectives" or "Learning Objectives" since that's what other  Armory tutorials use?

Reviewer: We probably should create a standard heading to use since I am also writing tutorials in CDaaS.

http://localhost:1313/contribute/create/guides/content-guide/#tutorial

- Create a Spinnaker application
- Create a Jenkins instance and add it to your application
- Deploy and manage a Spinnaker pipeline for your application
- Deploy updates to the application

> - Monitor the health of your application

Reviewer: '.' at the end of the bullet points since they are full sentences

> ## Prerequisites

Reviewer: ```suggestion
## {{% heading "prereq" %}}
```

> Before you begin make sure you have installed Quick Spin and have an active instance running on `locahost:9000` as described in: TBD.
> 
> - You need an active github account
> - Your favorite code editor with some JavaScript support enabled
> - 

Reviewer: ```suggestion
- Make sure you have installed Quick Spin and have an active instance running on `locahost:9000`. See {{< linkWithTitle "install-quick-spin.md" >}}.
- You have an active GitHub account.
- You have Javascript enabled in your favorite code editor.
```

## Run the Spinnaker prep pipeline

> ## Create kub cluster

Reviewer: ```suggestion
## Create a Kubernetes cluster
```
??  Is K8s included with Quick Spin? If not, having access to a K8s cluster should be listed in the `Before you begin` section.

> ## Get the tutorial repository:

Reviewer: ```suggestion
## Fork the tutorial repository
```

1. From your browser navigate to:
   ```

>    https://github.com/arnab-datta/counter-app

Reviewer: Great that you found this sample app. If you want to use Armory CD to deploy this sample app, you should fork it into github.com/armory and then reference that repo (so we own the repo).  
Are you going to "dockerize" this app and create K8s manifest files?

Author: Will get back to this one tomorrow

   ```

> 2. Create a fork to your personal github account.

Reviewer: ```suggestion
2. Create a fork to your personal GitHub account.
```
